test/t/base.sh

```diff
@@ -3,33 +3,27 @@
 set -uexo pipefail
 
 compilers=(
-    dmd-2.064
-    dmd-2.069.2
-    dmd-2.071.2
-    dmd-2.077.1
-    dmd-master-2018-10-14
-    ldc-1.4.0
+    dmd-2.088.1
```

Comment:
Because of Catalina, 2.088 is the minimum supported version.  Older releases of dmd fail to run due to missing functions errors in the dsym resolver.